### PR TITLE
Update versions and add cache url for redis.

### PR DESCRIPTION
**Pull request by nguni52**

### CODE REVIEW

File: pm4ml-experience-api/values.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ env:
   managementEndPoint: management-api
   dfspId: test
   metricsEndPoint: prometheus-server
+  CACHE_URL: redis://redis-master:6379
+  authSessionSecure: true
```
Comment (vijayg10) [inline pm4ml-experience-api/values.yaml:24]: I think these env variables are not going to be pulled in k8s deployment automatically. You may need to add these new parameters to pm4ml-experience-api/templates/deployment.yaml

Comment (nguni52) [reply pm4ml-experience-api/values.yaml:24]: Alright, noted.

File: pm4ml-experience-api/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
   - name: Murthy Kakarlamudi
     email: murthy@modusbox.com
+  - name: Nguni Phakela
+    email: nguni@izyane.com
+# dependencies:
+#   - name: redis
+#     version: 17.6.0
+#     repository: https://charts.bitnami.com/bitnami
+#     condition: redis.enabled
+
```
Comment (vijayg10) [inline pm4ml-experience-api/Chart.yaml:16]: You can remove the commented code here

File: pm4ml-experience-api/values.yaml [hunk 3]
```diff
@@ -20,6 +20,8 @@ env:
   managementEndPoint: management-api
   dfspId: test
   metricsEndPoint: prometheus-server
+  cacheUrl: redis://redis-master:6379
+  authSessionSecure: true
```
Comment (vijayg10) [inline pm4ml-experience-api/values.yaml:24]: Change the default value to false

File: pm4ml-experience-api/Chart.yaml (hunk 2 above)
Comment (nguni52) [reply pm4ml-experience-api/Chart.yaml:16]: Removed.

File: pm4ml-experience-api/values.yaml (hunk 3 above)
Comment (nguni52) [reply pm4ml-experience-api/values.yaml]: Done.

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**